src/components/app/service.ts

```diff
@@ -513,6 +537,16 @@ export function getTriggerHistory(pipelineId, params) {
     let URL = `${Routes.CI_CONFIG_GET}/${pipelineId}/workflows?offset=${params.offset}&size=${params.size}`
     return get(URL)
 }
+export function setImageTags(request, pipelineId, artifactId){
+    let URL = `${Routes.IMAGE_TAGGING}/${pipelineId}/${artifactId}`
+    return post(URL,request )
```

Comment:
we can directly use the url instead of creating variable.

Reply:
done